ambry-store/src/main/java/com.github.ambry.store/BlobStoreCompactor.java

```diff
@@ -49,6 +49,7 @@
  * the swap spaces.
  */
 class BlobStoreCompactor {
+  static final String COMPACTION_OPS_JOB_NAME = "compactionOps";
```

Comment:
`blob_store_compactor_log_read`?

Reply:
Actually, this is not log read, it's used in new log's write.
Maybe: 
```static final String COMPACTION_CLEANUP_JOB_NAME = "blob_store_compactor_cleanup";```

Similar to hard delete below.

Reply:
done.